src/index.html

```diff
@@ -12,7 +12,51 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="contain__stars">
```

Comment:
The class `contain__stars` is not part of the task description. Please revise this to align with the task guidelines.